sdk_container/src/third_party/coreos-overlay/coreos-base/misc-files/files/oems/azure

```diff
@@ -0,0 +1,13 @@
+/etc/systemd/system/nvidia.service
```

Comment:
I assume that the deletion happens after /etc is already overlaid? Also hopefully this won't delete third party files that accidentally had the same name.

Reply:
Yes, after the overlay. For user files with the same service name that we bring in I think the most practical solution is to document this because the same service name used will have other issues, too, I will add it to the changelog.